docs/tutorials/cargo.md

```diff
@@ -0,0 +1,119 @@
+# Setting up your Cargo.toml file for ICU4X
+
+ICU4X makes heavy use of small crates and Cargo features in order to be highly modular. This tutorial is intended to help you set up a Cargo.toml file to download what you need for ICU4X.
+
+## Basic Cargo.toml with testdata
+
+The most basic Cargo.toml to get you off the ground is the following:
+
+```toml
+[package]
+name = "demo"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[dependencies]
+icu = "1.0"
+icu_testdata = "1.0"
+```
+
+In your main.rs, you can use most functionality of ICU4X.
+
+[« Fully Working Example »](./cargo_tests/testdata)
+
+## Cargo.toml with experimental testdata
+
+If you wish to use an experimental feature, such as `icu_segmenter`, set up your Cargo.toml like this:
+
+```toml
+[package]
+name = "demo"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[dependencies]
+icu = { version = "1.0", features = ["experimental"] }
+icu_testdata = { version = "1.0", features = ["icu_segmenter"] }
+```
+
+In your main.rs, you can now use experimental features.
+
+[« Fully Working Example »](./cargo_tests/experimental)
+
+## Cargo.toml with Buffer Provider
+
+If you wish to generate your own data in blob format and pass it into ICU4X, enable the "serde" Cargo feature as follows:
+
+```toml
+[package]
+name = "demo"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[dependencies]
+icu = { version = "1.0", features = ["serde"] }
+icu_provider_blob = "1.0"
+```
+
+To generate your blob data, first run `cargo build` before you run `icu4x-datagen`.
+
+For more information on building ICU4X data, including whether to check in the postcard file to your repository, see [data_management.md](./data_management.md).
+
+[« Fully Working Example »](./cargo_tests/buffer)
+
+## Cargo.toml with Baked Provider and build.rs
+
+If you wish to use baked data for ICU4X, add some additional dependencies as follows:
+
+```toml
+[package]
+name = "demo"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[dependencies]
+icu = "1.0"
+icu_provider = "1.0" # for databake
+litemap = "0.6" # for databake
```

Comment:
litemap will be removed with 1.1

Reply:
(requires 1.1)

Reply:
All our tutorials are being continuously updated, doing it all at release time is a lot of work. Can we somehow release them instead of serving them from head?

Reply:
bump

Reply:
Done